### PR TITLE
Update simulators.rst

### PR DESCRIPTION
**Pull request by mmvandieren**
light copy edits as mentioned in #2207

### CODE REVIEW

File: docs/sphinx/using/backends/simulators.rst [hunk 1]
```diff
@@ -170,9 +170,9 @@ To execute a program on the multi-node multi-GPU NVIDIA target, use the followin
 
     .. note::
         (1) The order of the option settings are interchangeable.
-        For example, `cudaq.set_target('nvidia', option='mgpu,fp64')` is equivalent to `cudaq.set_target('nvidia', option='fp64.mgpu')`.
+        For example, `cudaq.set_target('nvidia', option='mgpu,fp64')` is equivalent to `cudaq.set_target('nvidia', option='fp64,mgpu')`.
 
-        (2) The `nvidia` target has single-precision as the default setting. Thus, using `option='mgpu'` implies that `option='mgpu,fp32'`.  
+        (1) The `nvidia` target has single-precision as the default setting. Thus, using `option='mgpu'` implies that `option='mgpu,fp32'`.
```
Comment (khalatepradnya) [inline docs/sphinx/using/backends/simulators.rst:175]: This renumbering doesn't appear to be right, screenshot from docs preview:
image

File: docs/sphinx/using/backends/simulators.rst [hunk 2]
```diff
@@ -169,6 +169,7 @@ To execute a program on the multi-node multi-GPU NVIDIA target, use the followin
     If a target is set in the application code, this target will override the :code:`--target` command line flag given during program invocation.
 
     .. note::
+
```
Comment (khalatepradnya) [inline docs/sphinx/using/backends/simulators.rst:172]: It is still the same... See: https://htmlpreview.github.io/?https%3A%2F%2Fraw.githubusercontent.com%2FNVIDIA%2Fcuda-quantum%2Fdocs%2Fpreview%2Fpr-2208%2Fusing%2Fbackends%2Fsimulators.html=

Shouldn't they be number serially, i.e., (1) and (2) or perhaps use bullets?

Comment (mmvandieren) [reply docs/sphinx/using/backends/simulators.rst:172]: Yes the original version was a hardcoding of the serial numbering but markdown didn't render that correctly, so I tried the markdown ordered numbering syntax and that didn't work either.  Your suggestion to just change it to bullet points is excellent. I'll add that commit.

**Comment by khalatepradnya** [comment]
image

**Review by khalatepradnya — APPROVED** [review]

**Pull request merged**